client/my-sites/site-settings/section-import.jsx

```diff
@@ -43,6 +43,7 @@ class SiteSettingsImport extends Component {
 
 	componentDidMount() {
 		ImporterStore.on( 'change', this.updateState );
+		this.updateFromAPI();
```

Comment:
should this be added to `componentDidUpdate` as well?

Reply:
Currently the status is updated every 5 seconds through the `<Interval />` component a little down in the code - [#](https://github.com/Automattic/wp-calypso/blob/c160ce777d1ee87ea3ec1de0ee1804f91fe2ca48/client/my-sites/site-settings/section-import.jsx#L139), so there is no need to change that for now. We can possibly find a better way to handle status updates when we refactor the code in the future.